mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeansModel.scala

```diff
@@ -42,10 +42,10 @@ class KMeansModel (@Since("1.0.0") val clusterCenters: Array[Vector],
   private[spark] val numIter: Int)
   extends Saveable with Serializable with PMMLExportable {
 
-  private val distanceMeasureInstance: DistanceMeasure =
+  @transient private lazy val distanceMeasureInstance: DistanceMeasure =
```

Comment:
Is the reason that this not serializable? if it's just to not serialize it, it might not be worth the overhead of lazy, but won't matter much. I can understand not serializing the cluster centers twice, effectively